syntaxes/vscode-autoit-output.tmLanguage.json

```diff
@@ -3,116 +3,167 @@
   "name": "vscode-autoit-output",
   "scopeName": "source.vscode_autoit_output",
   "patterns": [
-    {
-      "match": "^(#[0-9]+: | {4,})?((?:---|\\+\\+\\+).*)",
-      "captures":{
-        "1": {"name": "vscode-autoit-output-process-id"},
-        "2": {"name": "vscode-autoit-output-brown"}
-      }
+    {"include": "#header"}
+  ],
+  "repository": {
+    "header": {
+      "patterns": [
+        {
+          "comment": "Timestamp and/or process ID (12:34:56.789 #123: )",
+          "begin": "^(?:(\\d{2}:\\d{2}:\\d{2}\\.\\d{3})?( ?#\\d+:| {3,})? )?",
+          "end": "\\r?\\n|$",
```

Comment:
The header rule ends itself on a newline
so if nothing on same line pushes it out
then nothing will match on the next line

Reply:
that's what happens with `$` by itself